src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -146,6 +145,7 @@ the MIT License. See LICENSE in the project root for license information. -->
                     </Button>
                 </StackPanel>
             </ItemsControl>
+            <Button Grid.Row="4" Grid.Column="0" x:Uid="ColorScheme_DeleteButton" FontSize="15" Margin="0,0,0,20"/>

```

Comment:
I think this might want a margin left of 10? That or adjust all other left-most controls in this page to have a margin left of 0 🤔

Reply:
Good catch, missed that one